### ISSUE TITLE
Plans on supporting order: flexible?

### ISSUE THREAD

**Issue by cascornelissen** [hudochenkov/stylelint-order#8]
Are there any plans to support flexible ordering?

```
"order/declaration-block-properties-specified-order": [
    [
        "content",
        "position",
        "z-index",
        {
            "order": "flexible",
            "properties": ["top", "right", "bottom", "left"]
        },
        ...
    ]
]
```

See bullet point #2 for _array-of-unprefixed-property-names-or-group-objects_ in the [Stylelint documentation](https://stylelint.io/user-guide/rules/declaration-block-properties-order/#array-of-unprefixed-property-names-or-group-objects) for more info.

**Comment by hudochenkov** [comment]
I have no plans to add this rule. But I'm open for a PR with this rule. It should be named `declaration-block-properties-flexible-order`.

**Comment by cascornelissen** [comment]
I probably don't have time to do this myself in the next few weeks but I might be able to take a look at it later. If anyone else wants to work on this; go ahead!

For reference: https://github.com/stylelint/stylelint/blob/master/lib/rules/declaration-block-properties-order/index.js

**Comment by hudochenkov** [comment]
Stylelint's `declaration-block-properties-order` includes three types of order: strict, flexible, and alphabetical. This rule was dropped from stylelint core, because it was doing too much. So in this plugin we have more specified rules for different types of order. So this is the first point to have flexible order in a separate rule.

On the other side it will be a code duplication, because `flexible-order` apparently you want to have a mix of strict order and flexible order. It's bad to have almost whole rule duplicated.

One more thing to have in mind: `declaration-block-property-groups-structure` uses `declaration-block-properties-specified-order` inside. `property-groups-structure` converts its options to `properties-specified-order` options. I have no tests for this integration yet. I assume this integration will be broken by this PR.

To sum up: I have no strong opinion about how flexible order should be presented in this plugin. We have only two people opinions and it's not enough.
